pytorch_lightning/metrics/bleu.py

```diff
@@ -0,0 +1,143 @@
+from collections import Counter
+import math
+import re
+
+
+def math_log(num: int):
```

Comment:
can we torchify this by using torch.log?

Reply:
Okay, I will convert to Torch.